components/Filter/Index.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from "react";
```

Comment:
we don't need to import `React`

```suggestion
import { useState } from "react";
```